.github/workflows/acceptance_tests.yml

```diff
@@ -0,0 +1,64 @@
+name: Acceptance tests
```

Comment:
I think you need to rebase your branch on master...?

Reply:
Yes! Done.